docs/customization/localization.mdx

```diff
@@ -152,6 +152,22 @@ In the following example, the fr-FR locale is imported as `frFR`. The imported l
     })
     ```
   </Tab>
+
+  <Tab>
+    ```ts {{ filename: 'astro.config.mjs', mark: [3, 8] }}
```

Comment:
Should these be `js` rather than `ts`, based on the filename (`.mjs`)?

Although it may not make a significant difference right now we may differentiate between `js` and `ts` in the future, for example by displaying a small language icon next to the filename.

Reply:
Yeah let's start doing it! Updated 🫡